deployment/articles-dxp/01-saml/02-setting-up-identity-provider.markdown

```diff
@@ -0,0 +1,160 @@
+---
+header-id: setting-up-liferay-as-a-saml-identity-provider
+---
+
+# Setting up @product@ as a SAML Identity Provider
+
+An identity provider is a trusted provider that provides single sign-on for
+users to access other websites. A service provider is a website that hosts
+applications and grants access only to identified users with proper credentials.
+Liferay Portal 6.1 EE and later versions support SAML 2.0 integration via the 
+[Liferay SAML 2.0 Provider](https://web.liferay.com/marketplace/-/mp/application/15188711)
+application. It is provided from Liferay Marketplace and allows @product@ to act
+as a SAML 2.0 identity provider or as a service provider.
+**Important:** You can set @product@ up as an Identity Provider or as a Service Provider. Each single @product@ instance can serve as an identity provider or as a
```

Comment:
Suggest: Can we move the "important" block the a "note" block to better highlight it?

Reply:
Yup. Done.